src/main/kotlin/app/revanced/patches/all/ads/bytecode/AdsBytecodePatch.kt

```diff
@@ -0,0 +1,77 @@
+package app.revanced.patches.all.ads.bytecode
+
+import app.revanced.patcher.annotation.*
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.util.bytecode.*
+import app.revanced.patches.all.ads.blocklist.*
+import app.revanced.patcher.util.proxy.mutableTypes.MutableClass
+import app.revanced.patcher.util.proxy.mutableTypes.MutableMethod
+import org.jf.dexlib2.Opcode
+import org.jf.dexlib2.iface.instruction.formats.Instruction21c
+import org.jf.dexlib2.iface.instruction.formats.Instruction35c
+import org.jf.dexlib2.iface.reference.MethodReference
+import org.jf.dexlib2.iface.reference.StringReference
+import org.jf.dexlib2.immutable.reference.ImmutableStringReference
+import org.jf.dexlib2.iface.ClassDef
+import org.jf.dexlib2.iface.Method
+import org.jf.dexlib2.builder.instruction.BuilderInstruction21c
+import org.jf.dexlib2.builder.instruction.BuilderInstruction10x
+
+@Patch(false)
+@Name("ads")
+@Description("Removes all ads from the selected application.")
+@Version("0.0.1")
+class AdsBytecodePatch : BytecodePatch() {
+    override fun execute(context: BytecodeContext): PatchResult {
+        BytecodeUtils.transformIns(context, ::invokesTransformFunction, invokeOpcodes)
+        BytecodeUtils.transformIns(context, ::urlTransformFunction, stringOpcodes)
+
+        return PatchResultSuccess()
+    }
+
+    private fun invokesTransformFunction(Ins: Any, index: Int, methodDef: Method, classDef: ClassDef, context: BytecodeContext) {
+        if (Ins is Instruction35c) {
+            val instructionMethodReference = Ins.getReference() as MethodReference
+
+            if (blockInvokes.any{blockInvokes.contains(instructionMethodReference.getName())}) {
```

Comment:
This seems odd to me. Why would you use `contains` in `any`? Probably you wanted:

```suggestion
            if (blockInvokes.any{ it == instructionMethodReference.getName() }) {
```